test/jdk/java/text/Format/DecimalFormat/LargeExponentsTest.java

```diff
@@ -49,14 +49,16 @@ public class LargeExponentsTest {
     // Exponent symbol is 'E'
     private static final NumberFormat FMT = NumberFormat.getInstance(Locale.US);
 
-    // Check that the parsed value is equal to the expected.
-    // We are mainly checking that an exponent > Integer.MAX_VALUE no longer parses to 0
-    // and that an exponent > Long.MAX_VALUE no longer parses to the mantissa.
+    // Check that the parsed value and parse position index are both equal to the expected values.
+    // We are mainly checking that an exponent > Integer.MAX_VALUE no longer
+    // parses to 0 and that an exponent > Long.MAX_VALUE no longer parses to the mantissa.
     @ParameterizedTest
     @MethodSource({"largeExponentValues", "smallExponentValues", "bugReportValues", "edgeCases"})
-    public void overflowTest(String parseString, Double expectedValue) throws ParseException {
-        Number actualValue = FMT.parse(parseString);
+    public void overflowTest(String parseString, Double expectedValue) {
```

Comment:
Since this is a regression test, it may be better having both 1-arg parse() and 2-arg parse() tested separately, instead of replacing.